choco/nunit-console.portable.nuspec

```diff
@@ -0,0 +1,52 @@
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nunit-console.portable</id>
+    <version>3.6.1</version>
+    <title>NUnit Console Runner Version 3 (No Extensions, Portable)</title>
+    <summary>Console runner for the NUnit 3 unit-testing framework, without any extensions.</summary>
+    <description>
+      This package includes the nunit3-console runner and test engine for version 3 of the NUnit unit-testing framework.
+    </description>
+    <projectUrl>http://nunit.org</projectUrl>
+    <projectSourceUrl>https://github.com/nunit/nunit-console</projectSourceUrl>
+    <docsUrl>https://github.com/nunit/docs/wiki/Console-Command-Line</docsUrl>
+    <bugTrackerUrl>https://github.com/nunit/nunit-console/issues</bugTrackerUrl>
+    <iconUrl>https://cdn.rawgit.com/nunit/resources/master/images/icon/nunit_256.png</iconUrl>
+    <packageSourceUrl>https://github.com/nunit/nunit-console</packageSourceUrl>
+	<releaseNotes>https://raw.githubusercontent.com/nunit/nunit-console/master/CHANGES.txt</releaseNotes>
+	<mailingListUrl>https://groups.google.com/forum/#!forum/nunit-discuss</mailingListUrl>
+    <licenseUrl>http://nunit.org/nuget/nunit3-license.txt</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <authors>nunit</authors>
+    <owners>nunit</owners>
+	<language>en-US</language>
+    <tags>nunit console runner test testing tdd</tags>
+    <copyright>Copyright (c) 2017 nunit</copyright>
+  </metadata>
+  <files>
+    <file src="LICENSE.txt" />
+    <file src="NOTICES.txt" />
+    <file src="CHANGES.txt" />
+    <file src="bin\nunit-agent.exe" target="tools" />
+    <file src="bin\nunit-agent.exe.config" target="tools" />
+    <file src="bin\nunit-agent-x86.exe" target="tools" />
+    <file src="bin\nunit-agent-x86.exe.config" target="tools" />
+    <file src="bin\nunit3-console.exe" target="tools" />
+    <file src="bin\nunit3-console.exe.config" target="tools" />
+    <file src="bin\nunit.engine.api.dll" target="tools" />
+    <file src="bin\nunit.engine.api.xml" target="tools" />
+    <file src="bin\nunit.engine.dll" target="tools" />
+    <file src="bin\Mono.Cecil.dll" target="tools" />
+    <file src="..\..\nuget\engine\nunit.nuget.addins" target="tools"/>
+	<file src="addins\nunit-project-loader.dll" target="tools\addins" />
+	<file src="addins\nunit.v2.driver.dll" target="tools\addins" />
+	<file src="addins\nunit.core.dll" target="tools\addins" />
+	<file src="addins\nunit.core.interfaces.dll" target="tools\addins" />
+	<file src="addins\nunit.v2.driver.addins" target="tools\addins" />
+	<file src="addins\nunit-v2-result-writer.dll" target="tools\addins" />
+	<file src="addins\teamcity-event-listener.dll" target="tools\addins" />
+	<file src="addins\vs-project-loader.dll" target="tools\addins" />
+  </files>
```

Comment:
I'm not up on the details, but it's done in the nunit-distribution project. However, I think it's all pretty simple if you inspect what you are getting from each nuget package you read. You can either create a directory structure that matches the structure implied by the `.addins` file, or not use them and create new files of your own that match the structure you are creating.

In your case, you have a simple structure with the engine under tools and the extensions in tools/addins. Therefore to match that, the .addins file in tools should contain `addins/` and the file or files in addins should contain entries like: `nunit-project-loader.dll`.

That's not what you have. The file `nunit.nuget.addins` that is part of the engine nuget install contains some complex paths that allow it to use extensions installed side-by-side with it in other nuget packages rather than the simple reference to `addins` that you need. Except for the v2 framework driver, the individual extensions don't have `.addins` files. When installed by themselves, they don't need one because they consist of a single file, all by itself in the tools directory. You need to create your own .addins file that contains all the assemblies you want examined for extensions. That list is easy: it's one dll out of each of the extensions you load. All the extensions except for the v2 framework driver have just one dll, so that's easy enough. For an initial implementation, you could just add the .addins file to the project. Eventually, we should generate it from a list of desired extensions.